frontend/src/app/search.tsx

```diff
@@ -98,6 +125,60 @@ export default class Search extends Component {
     );
   }
 
+  private handleChange = (checked, event) => {
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    const name = target.name;
+    this.setState({ [name]: value }, ()=> {
+      if(checked === true){
+        this.setState({count: this.state.count+1}, () => {
+          if(this.state.count === 0){
+            this.setState({show: false})
+          }else{
+            this.setState({show: true})
+          }
+        })
+        this.tpaths.push(name);   
+        console.log('tpaths:'+this.tpaths)
+      }else{
+        this.setState({count: this.state.count-1}, () => {
+          if(this.state.count === 0){
+            this.setState({show: false})
+          }else{
+            this.setState({show: true})
+          }
+        })
+        this.tpaths.splice(this.tpaths.indexOf(name))
+        console.log('tpaths:'+this.tpaths)
+      }
+    });
+  };
+
+  delete = (event, keydata) => {
+    console.log('in the delete function')
+    for(var i=0;i<keydata.length;i++){
+      // console.log(keydata[i])
+      const formData = new FormData();
+      formData.append('module', keydata[i])
+      fetch('/content/'+keydata[i], {
+        method: 'post',
+        body: formData
+      }).then(response => {
+        if (response.status == 200) {
+          console.log(" Delete Works " + response.status)
+          this.state.flag = true
+          //window.location.href = "/pantheon"
+        } else if (response.status == 403) {
+          this.setState({ authMessage: "Delete failed." })
```

Comment:
Again, we already talked about it, but just to write it here... 1.) don't use authMessage, that's for a separate feature, and 2.) use a patternfly widget instead of a javascript alert or console.log.

Reply:
Acknowledged.
Thanks for the suggestions, and going forward, I will take note of the mentioned points.